resources/git-hooks/commit-msg

```diff
@@ -106,11 +109,18 @@ function print_error() {
   echo -e "Valid scopes: ${t_grn}${scopes[@]}${t_end}"
   echo -e "Max length (first line): ${t_grn}$max_length${t_end}"
   echo -e "Min length (first line): ${t_grn}$min_length${t_end}"
-  echo -e "Optionally, append ${t_grn}Fixes #1234${t_end}\n"
-  echo -e "${t_cyn}Example:${t_end} fix(windows): Re-attaches the widget plug which had fallen out. Fixes #1111"
+  echo -e "If possible, append git trailers:"
+  echo -e " * ${t_grn}Fixes: #1234${t_end}"
+  echo -e " * ${t_grn}Fixes: KEYMAN-MODULE-XYZ${t_end}"
+  echo -e " * ${t_grn}Cherry-pick-of: #2468${t_end}"
```

Comment:
Just did #11615 and used the "Cherry-pick-of:" trailer - our auto-labeler didn't auto-label the resulting PR as a cherry-pick, though.

Branch name also included a `/cherry/` component.

Is there anything in the works to auto-apply that label?

Reply:
Currently, only the branch name is checked -- not git trailers. And it should be `/cherry-pick/` not `/cherry/`. See https://github.com/keymanapp/keyman/wiki/Pull-Request-and-Commit-workflow-notes#branch-names

Reply:
It looks like... when I manually add the 🍒 emoji, the rest triggers?

Reply:
Yes, adding either the emoji causes the bot to add the label, and I think vice-versa works too. But alas the branch name is immutable.

Reply:
> Currently, only the branch name is checked -- not git trailers. And it should be `/cherry-pick/` not `/cherry/`. See https://github.com/keymanapp/keyman/wiki/Pull-Request-and-Commit-workflow-notes#branch-names

Huh, I've been using `/cherry/` for years, and it never really came up.

Reply:
It's only used for auto labelling at this point